src/content/quickstart/kind/_index.md

```diff
@@ -0,0 +1,18 @@
+---
+date: 2020-03-17T13:36:18+01:00
+title: "KIND"
+---
+
+## KIND
+
+[KIND](https://github.com/kubernetes-sigs/kind) is a tool to run local Kubernetes clusters inside Docker container nodes.
+
+Submariner provides scripts that deploy 3 Kubernetes clusters locally, 1 broker and 2 data clusters with Submariner dataplane components deployed on both the data clusters.
```

Comment:
The Submariner components are actually deployed on all the clusters so technically this should read:

```suggestion
Submariner provides scripts that deploy 3 Kubernetes clusters locally, 1 broker and 2 data clusters with the Submariner dataplane components deployed on all the clusters.
```